### PR TITLE
Add documentation for dotnet nuget config commands

### LINKED ISSUE #13398: Document dotnet nuget config
This may need to be something that lives in the dotnet/docs repo instead.

### PR DESCRIPTION
**Pull request by martinrrm**
## Summary

Add documentation for `dotnet nuget config` commands.
Implementation: https://github.com/NuGet/NuGet.Client/pull/5549

Fixes https://github.com/NuGet/Home/issues/13398

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/tools/dotnet-nuget-config-get.md](https://github.com/dotnet/docs/blob/cba033dc2f6705418ab16bb47f6cce532ec2be3f/docs/core/tools/dotnet-nuget-config-get.md) | [dotnet nuget config get](https://review.learn.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-config-get?branch=pr-en-us-41019) |
| [docs/core/tools/dotnet-nuget-config-paths.md](https://github.com/dotnet/docs/blob/cba033dc2f6705418ab16bb47f6cce532ec2be3f/docs/core/tools/dotnet-nuget-config-paths.md) | [dotnet nuget config paths command](https://review.learn.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-config-paths?branch=pr-en-us-41019) |
| [docs/core/tools/dotnet-nuget-config-set.md](https://github.com/dotnet/docs/blob/cba033dc2f6705418ab16bb47f6cce532ec2be3f/docs/core/tools/dotnet-nuget-config-set.md) | [dotnet nuget config set](https://review.learn.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-config-set?branch=pr-en-us-41019) |
| [docs/core/tools/dotnet-nuget-config-unset.md](https://github.com/dotnet/docs/blob/cba033dc2f6705418ab16bb47f6cce532ec2be3f/docs/core/tools/dotnet-nuget-config-unset.md) | [docs/core/tools/dotnet-nuget-config-unset](https://review.learn.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-config-unset?branch=pr-en-us-41019) |

### CODE REVIEW

**Review by tdykstra — COMMENTED** [review]
Several markdownlint errors need to be addressed:
Error: docs/core/tools/dotnet-nuget-config-get.md:40:1 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
Error: docs/core/tools/dotnet-nuget-config-get.md:65:62 MD047/single-trailing-newline Files should end with a single newline character
Error: docs/core/tools/dotnet-nuget-config-paths.md:27 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
Error: docs/core/tools/dotnet-nuget-config-paths.md:52:62 MD047/single-trailing-newline Files should end with a single newline character
Error: docs/core/tools/dotnet-nuget-config-set.md:25:30 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
Error: docs/core/tools/dotnet-nuget-config-set.md:61:62 MD047/single-trailing-newline Files should end with a single newline character
Error: docs/core/tools/dotnet-nuget-config-unset.md:25:32 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
Error: docs/core/tools/dotnet-nuget-config-unset.md:33 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
Error: docs/core/tools/dotnet-nuget-config-unset.md:58:62 MD047/single-trailing-newline Files should end with a single newline character

File: docs/core/tools/dotnet-nuget-config-unset.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+## Examples
+
+* Removes's the `repositoryPath` config value from the current directory:
+
+  ```dotnetcli
+  dotnet nuget config unset repositoryPath
+  ```
+
+* Removes's the `repositoryPath` config value from the specified configuration file:
```
Comment (tdykstra) [inline docs/core/tools/dotnet-nuget-config-unset.md:50]:
```suggestion
* Removes the `repositoryPath` config value from the current directory:

  ```dotnetcli
  dotnet nuget config unset repositoryPath
  ```

* Removes the `repositoryPath` config value from the specified configuration file:
```

**Review by tdykstra — COMMENTED** [review]
Looks good. I left a few suggestions for minor fixes.

**Comment by BillWagner** [comment]
Hi @martinrrm 

Can you address the review comments?

**Review by tdykstra — APPROVED** [review]
LGTM

**Comment by gewarren** [comment]
Oops, these weren't added to the TOC.

**Pull request merged**